src/coreclr/src/gc/env/gcenv.os.h

```diff
@@ -290,7 +290,7 @@ class GCToOSInterface
     //  size    - size of the virtual memory range
     // Return:
     //  Address of the allocated memory
-    static void* VirtualReserveAndCommitLargePages(size_t size);
+    static void* VirtualReserveAndCommitLargePages(size_t size, uint16_t node = NUMA_NODE_UNDEFINED);
```

Comment:
since you are changing this you'll need to change the linux impl as well in src\gc\unix\gcenv.unix.cpp

Reply:
Yes, good point. Will do.